server/tests/security/test_command_safety_runtime.py

```
@@ -0,0 +1,344 @@
+    check_command_safety,
+)
+
+_HEX_ALPHABET = set("0123456789abcdef")
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _patch_config(monkeypatch, *, enabled: bool):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_patch_config`

Add return type annotation: `None`

(ANN202)